prisma-fmt/tests/hover/scenarios/model_from_model_type_on_broken_relations/schema.prisma

```diff
@@ -0,0 +1,21 @@
+datasource db {
+    provider = "postgresql"
+    url      = env("DATABASE_URL")
+}
+
+model interm {
+    id Int @id
+
+    forumId Int
+    forum   For<|>um @relation(fields: [forumId], references: [id])
+}
+
+/// ! Commenting this yields a panic
```

Comment:
I don't think we need this comment or commented out model anymore, panic is gone, all works as exepcted.